tools/ci-build/scripts/generate-aws-sdk

```diff
@@ -10,11 +10,12 @@ C_RESET='\033[0m'
 set -eux
 
 echo -e "${C_YELLOW}Taking examples from 'awsdocs/aws-doc-sdk-examples'...${C_RESET}"
+examples_revision=$(cd aws-doc-sdk-examples; git rev-parse HEAD)
```

Comment:
shellcheck would probably want quotes here?

Reply:
Surprisingly not in this case...